test/fury.js

```diff
@@ -223,7 +349,7 @@ describe('Refract loader', () => {
       annotation = result.get(1);
     });
 
-    context('parse result annotation', () => {
+    context('and parse result annotation', () => {
```

Comment:
Nitpick: I think this should be something different, along with line 342. IMO, it should be:

```js
// line 342
context('when given a valid API'. function() {
  // All `it` assertions here
});
```

The "and" feels off to me.